src/Caching/StackExchangeRedis/src/StackExchangeRedisCacheServiceCollectionExtensions.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using Microsoft.Extensions.Caching.Distributed;
 using Microsoft.Extensions.Caching.StackExchangeRedis;
+using Microsoft.Extensions.DependencyInjection.Extensions;
```

Comment:
Shouldn't be needed?

Reply:
It's needed. TryAdd method needs it which is used on line 30.